.circleci/config.yml

```diff
@@ -4052,39 +4048,24 @@ jobs:
-
-            DOCS_TAG=$(make --quiet docs-tag)
-            MAIN_TAG=$(make --quiet tag)
-
-            # The docs image uses the same tag as main for releases,
-            # but is built as part of the main build with a different tag.
-            # This step places it in Docker Hub with the release tag
-            # for convenience before running the stackrox.io push.
-            ./scripts/ci/pull-retag-push.sh  "docker.io/stackrox/docs:$DOCS_TAG" "docker.io/stackrox/docs:$MAIN_TAG"
-
       - run:
           name: Push image into stackrox.io
           command: |
-            docker login -u "$DOCKER_IO_PUSH_USERNAME" --password-stdin \<<<"$DOCKER_IO_PUSH_PASSWORD" docker.io
             docker login -u "$STACKROX_IO_PUSH_USERNAME" --password-stdin \<<<"$STACKROX_IO_PUSH_PASSWORD" stackrox.io
             docker login -u "$STACKROX_IO_PUSH_USERNAME" --password-stdin \<<<"$STACKROX_IO_PUSH_PASSWORD" collector.stackrox.io
 
             MAIN_TAG=$(make --quiet tag)
             COLLECTOR_TAG=$(make --quiet collector-tag)
-            ./scripts/ci/pull-retag-push.sh "docker.io/stackrox/main:$MAIN_TAG" "stackrox.io/main:$MAIN_TAG"
+            ./scripts/ci/pull-retag-push.sh "quay.io/stackrox-io/main:$MAIN_TAG" "stackrox.io/main:$MAIN_TAG"
 
             # The docs image uses the same tag as main for releases.
-            ./scripts/ci/pull-retag-push.sh "docker.io/stackrox/docs:$MAIN_TAG"   "stackrox.io/docs:$MAIN_TAG"
-            ./scripts/ci/pull-retag-push.sh "docker.io/stackrox/roxctl:$MAIN_TAG" "stackrox.io/roxctl:$MAIN_TAG"
-            ./scripts/ci/pull-retag-push.sh "docker.io/stackrox/central-db:$MAIN_TAG" "stackrox.io/central-db:$MAIN_TAG"
+            ./scripts/ci/pull-retag-push.sh "quay.io/stackrox-io/docs:$MAIN_TAG"   "stackrox.io/docs:$MAIN_TAG"
+            ./scripts/ci/pull-retag-push.sh "quay.io/stackrox-io/roxctl:$MAIN_TAG" "stackrox.io/roxctl:$MAIN_TAG"
+            ./scripts/ci/pull-retag-push.sh "quay.io/stackrox-io/central-db:$MAIN_TAG" "stackrox.io/central-db:$MAIN_TAG"
 
-            ./scripts/ci/pull-retag-push.sh "docker.io/stackrox/collector:${COLLECTOR_TAG}"        "collector.stackrox.io/collector:${COLLECTOR_TAG}"
-            ./scripts/ci/pull-retag-push.sh "docker.io/stackrox/collector:${COLLECTOR_TAG}-latest" "collector.stackrox.io/collector:${COLLECTOR_TAG}-latest"
-            ./scripts/ci/pull-retag-push.sh "docker.io/stackrox/collector:${COLLECTOR_TAG}-slim"   "collector.stackrox.io/collector:${COLLECTOR_TAG}-slim"
+            ./scripts/ci/pull-retag-push.sh "quay.io/stackrox-io/collector:${COLLECTOR_TAG}"        "collector.stackrox.io/collector:${COLLECTOR_TAG}"
+            ./scripts/ci/pull-retag-push.sh "quay.io/stackrox-io/collector:${COLLECTOR_TAG}-latest" "collector.stackrox.io/collector:${COLLECTOR_TAG}-latest"
+            ./scripts/ci/pull-retag-push.sh "quay.io/stackrox-io/collector:${COLLECTOR_TAG}-slim"   "collector.stackrox.io/collector:${COLLECTOR_TAG}-slim"
```

Comment:
All of these under `Push image into stackrox.io` step should have `quay.io/rhacs-eng` instead of `quay.io/stackrox-io` as source. `/rhacs-eng` gets commercially branded images unlike `/stackrox-io`

Reply:
Done, thx